cpp/test/sg/rf_treelite_test.cu

```diff
@@ -416,14 +415,14 @@ class RfConcatTestReg : public RfTreeliteTestCommon<T, L> {
 
     for (int i = 0; i < 3; i++) {
       ModelHandle model;
-      std::vector<unsigned char> vec_data;
+
       this->rf_params.n_trees = this->rf_params.n_trees + i;
 
       fit(*(this->handle), this->all_forest_info[i], this->data_d,
           this->params.n_rows, this->params.n_cols, this->labels_d,
           this->rf_params);
       build_treelite_forest(&model, this->all_forest_info[i],
-                            this->params.n_cols, this->task_category, vec_data);
+                            this->params.n_cols, this->task_category);
```

Comment:
I like this new style without the old vec_data.

Reply:
Same. I also like how `build_treelite_forest()` is now doing only one job.